docs/de-DE/config/README.md

```diff
@@ -1,1302 +0,0 @@
-# Konfiguration
```

Comment:
I don't think you should delete all the readme files.

Reply:
Sorry, messed up... :-( misunderstood what what the delete in the PR is doing... 
I will find a way to fix that.

Reply:
Download a clean repo, copy the readme files and commit them.

Reply:
k